### PR TITLE
fix: Audio tool clipRegion loading display

### PR DESCRIPTION
**Pull request by lixinghua123**

### CODE REVIEW

File: packages/lb-components/src/components/audioPlayer/index.tsx [hunk 1]
```diff
@@ -808,6 +811,28 @@ export const AudioPlayer = ({
     setAudioUrl();
   }, [url]);
 
+  useEffect(() => {
+    getVisibleAreaRange();
+  }, [waveSize, duration]);
+
+  const getVisibleAreaRange = () => {
```
Comment (Glenfiddish) [inline packages/lb-components/src/components/audioPlayer/index.tsx:818]: side effect event should not name as getter

File: packages/lb-components/src/components/audioPlayer/zoomSlider/index.tsx [hunk 2]
```diff
@@ -9,13 +9,14 @@ import { useLatest } from 'ahooks';
...
 
-const EKeyCode = cKeyCode.default
+const EKeyCode = cKeyCode.default;
 
 // 建议用户裁剪音频到10分钟以下送标
 export const audioZoomInfo = {
   min: 1,
   max: 150,
-  ratio: 1,
+  ratio: 10,
```
Comment (Glenfiddish) [inline packages/lb-components/src/components/audioPlayer/zoomSlider/index.tsx:18]: is 10 suitable here? Why not 2 or 4 or 8?

File: packages/lb-components/src/components/audioPlayer/index.tsx [hunk 3]
```diff
@@ -808,6 +811,28 @@ export const AudioPlayer = ({
     setAudioUrl();
   }, [url]);
 
+  useEffect(() => {
+    getVisibleAreaRange();
+  }, [waveSize, duration]);
```
Comment (Glenfiddish) [inline packages/lb-components/src/components/audioPlayer/index.tsx:816]: Will it cause pool performance if change too quickly?

Comment (lixinghua123) [reply packages/lb-components/src/components/audioPlayer/index.tsx:816]: Not changing quickly

**Pull request merged**